emyjamalian/todo
# E2e feb

This PR is for 
- Updating the e2e tests to make them work on main and also I made sure it's not false positive or false negative.
- Adding the e2e tests to CI pipeline so now with every new PR we test the main functionalities :) 

------------------------------------
Next steps: 
use best practice of page object model for the test

------------------------------------
How to test: 
- You can checkout the branch locally and make sure the tests will pass as many times as you run them.
- You can create a subbranch and commit different things to check the Github Actins working as expected :) 

------------------------------------


P.S. This felt so good 🎉🥳
